### DISCUSSION TITLE
Download a folder

### DISCUSSION THREAD

**Discussion by Jamiecharris** [supabase/supabase#10287] [Questions]
Can you generate a download link for a folder in a bucket? 

If not what would be a work around?

**Comment by GaryAustin1** [comment]
Files have to be downloaded one at a time.  You would need to use .list() with a path to the folder to get all files then loop and download them.

https://supabase.com/docs/reference/javascript/storage-from-list

 You can pass createSignedUrls an array of pathnames.
